### PR TITLE
Allow Redux Dev Tools Options

### PR DESCRIPTION
**Pull request by rrex-godaddy**
## Summary

This PR is to allow Devtools options to be passed to the gasket redux plugin

### CODE REVIEW

File: packages/gasket-redux/src/configure-make-store.js [hunk 1]
```diff
@@ -86,7 +87,9 @@ export default function configureMakeStore(
       );
     }
 
-    const composer = (typeof window !== 'undefined' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose;
+    const composer = (typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) ?
+      window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ ...devToolsOptions }) : compose;
```
Comment (jpage-godaddy) [inline packages/gasket-redux/src/configure-make-store.js:91]: Unnecessary spread

```suggestion
      window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__(devToolsOptions) : compose;
```

File: packages/gasket-redux/src/configure-make-store.js [hunk 2]
```diff
@@ -44,6 +44,7 @@ export function prepareReducer(allReducers, rootReducer) {
  */
 export default function configureMakeStore(
   {
+    devToolsOptions = {},
```
Comment (jpage-godaddy) [inline packages/gasket-redux/src/configure-make-store.js:47]: Could you document this new option in the README.md for `@gasket/redux`?

Comment (agerard-godaddy) [reply packages/gasket-redux/src/configure-make-store.js:47]: @rrex-godaddy is this PR still desirable? If so, could you please address the two items so we can get it merged and published?

**Comment by mmason2-godaddy** [comment]
Reached out to @rrex-godaddy on this

**Comment by agerard-godaddy** [comment]
Closing due to inactivity

**Pull request closed without merging**